samples/test/metrics_visualization_v2.py

```diff
@@ -89,7 +89,7 @@ def wine_classification(metrics: Output[ClassificationMetrics]):
     y_predict = cross_val_predict(rfc, X_train, y_train, cv=3, method='predict')
```

Comment:
What do you think about modifying the inf in a way that maintains its meaning, rather than omitting? I defer to whatever your preference is here.

```python
    # avoid inf thresholds
    epsilon = 1e-6
    thresholds = [1 - epsilon if t == float('inf') else t for t in thresholds]
```